conventions/src/main/kotlin/otel.bom-conventions.gradle.kts

```diff
@@ -33,3 +33,21 @@ afterEvaluate {
     }
   }
 }
+
+evaluationDependsOn(":dependencyManagement")
+val dependencyManagementConf = configurations.create("dependencyManagement") {
+  isCanBeConsumed = false
+  isCanBeResolved = false
+  isVisible = false
+}
+afterEvaluate {
+  configurations.configureEach {
+    if (isCanBeResolved && !isCanBeConsumed) {
+      extendsFrom(dependencyManagementConf)
+    }
+  }
+}
+
+dependencies {
+  add(dependencyManagementConf.name, platform(project(":dependencyManagement")))
```

Comment:
Just to make 100% sure: will it cause all the things in our dependency management bom (spring, slf4j version etc) to be added to the OTel bom?

Reply:
no, this is just enforcing versions of dependencies that are used, copy-pasted from `otel.java-conventions.gradle.kts`.

and I just triple checked that it doesn't bring anything extra into the published bom 👍